addons/core/tests/integration/helpers/app-name-test.js

```diff
@@ -12,7 +12,7 @@ module('Integration | Helper | app-name', function (hooks) {
   setupRenderingTest(hooks);
 
   test('it renders', async function (assert) {
-    await render(hbs`{{app-name}}`);
+    await render(hbs`<app-name />`);
```

Comment:
This and the 2 other test files below where [changed due to this eslint rule](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-implicit-this.md). The recommendation from the codemod is to tuse angle bracket when its a helper with no arguments.

Reply:
@aklkv 
You are totally right. After commiting this I realize I miss understood the recomendation, recomendation to use angle bracket was for component but this is a helper with no arguments, therefor make sense to use them with classic invocation.
I opted for adding [the eslint rule exempion ](https://github.com/hashicorp/boundary-ui/pull/2777/files#diff-543650cb72a7d2bc2e89a5d88c4b4a346c39869569c9e4fd52b08c8598a57d78R15), thanks for the catch!! 🙌